lib/detect-system.sh

```diff
@@ -31,7 +31,7 @@ parse_executable_arch()
 			_system_arch=aarch64;;
 		*ARM)
 			_system_arch=arm;;
-		 *RISC_V)
+		 *RISC-V)
```

Comment:
thanks, this change can be amended to early code which add RISC V in parse_executable_arch, so the patchset doesn't require to have fix for itself.

Reply:
@rli9 I am not able to understand. Could you please elaborate?
Thanks

Reply:
You can rebase the whole patchset, so there's no intermediate fix/change for the patchset itself since the early commits are not applied yet. For example, patch 1 uses RISC_V, then patch 2 switches to RISC-V, these 2 can be merged to single patch which uses RISC-V at the beginning.